packages/nvidia_nat_phoenix/src/nat/plugins/phoenix/scripts/export_trajectory_to_phoenix/atif_trajectory_exporter.py

```
@@ -0,0 +1,694 @@
+
+            if source == "user":
+                msg = _message_to_str(step.get("message", ""))
+                if first_user_msg is None and msg:
+                    first_user_msg = msg
+                continue
+
+            # Agent or system steps — need valid AtifStepExtra
+            extra_raw = step.get("extra") or {}
+            try:
+                step_extra = AtifStepExtra.model_validate(extra_raw)
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)